website/docs/enterprise/getting-started/install-enterprise.mdx

```diff
@@ -31,70 +31,53 @@ import TOCInline from "@theme/TOCInline";
 There is no need to install the open source version of Weave GitOps before installing Weave GitOps Enterprise.
 :::
 
-## Example: Set up a Management Cluster with CAPA and EKS
+## Prerequisites
 
-To get you started, we'll cover EKS as our management cluster with the CAPA provider. Please note again that Weave GitOps Enterprise supports [clusters without Cluster API](../../cluster-management/managing-clusters-without-capi.mdx), as well as any combination of management cluster and CAPI provider.
+To get up and running with Weave GitOps Enterprise:
+- create a Kubernetes cluster
+- install Weave GitOps Enterprise, using the `gitops-ee bootstrap` command
```

Comment:
I'm not sure if we have this command in the EE `gitops` command (would be cool though!)

Reply:
@foot I know what happened ... removing that command language. Thanks for the catch. :)